platform/android/android_keys_utils.h

```diff
@@ -60,6 +60,7 @@ static AndroidGodotCodePair android_godot_code_pairs[] = {
 	{ AKEYCODE_DPAD_DOWN, Key::DOWN }, // (20) Directional Pad Down key.
 	{ AKEYCODE_DPAD_LEFT, Key::LEFT }, // (21) Directional Pad Left key.
 	{ AKEYCODE_DPAD_RIGHT, Key::RIGHT }, // (22) Directional Pad Right key.
+	{ AKEYCODE_DPAD_CENTER, Key::DPAD_CENTER }, // (23) Directional Pad Center key.
```

Comment:
Seems like we should be able to reuse `Key::ENTER` for `AKEYCODE_DPAD_CENTER` instead?

That would mirror what we are doing for the other dpad keycodes.

Reply:
Especially unless we add it to standard actions, I'd expect the controller to be able to trigger `ui_accept` and such